src/data/real/ennreal.lean

```diff
@@ -1094,6 +1103,9 @@ begin
   refine (le_div_iff_mul_le _ _).symm; simpa
 end
 
+lemma lt_div_iff_mul_lt (hb0 : b ≠ 0 ∨ c ≠ ∞) (hbt : b ≠ ∞ ∨ c ≠ 0) : c < a / b ↔ c * b < a :=
+by simp only [← not_le, div_le_iff_le_mul hb0 hbt]
```

Comment:
I think you can use `lt_iff_lt_of_le_iff_le` here.

Reply:
Done, thanks!